vmm/src/cpu.rs

```diff
@@ -1176,13 +1177,14 @@ impl CpuManager {
                                 vcpu_run_interrupted.store(true, Ordering::SeqCst);
 
                                 vcpu_paused.store(true, Ordering::SeqCst);
-                                while vcpu_pause_signalled.load(Ordering::SeqCst) {
+                                while vcpus_pause_signalled.load(Ordering::SeqCst) {
                                     thread::park();
                                 }
+                                vcpu_paused.store(false, Ordering::SeqCst);
```

Comment:
btw: this is the line were the individual vCPU tells the cPU manager is has been resumed